tests/jax/test_fused_attn.py

```diff
@@ -174,27 +189,62 @@ def _check_configs(self):
         if self.qkv_layout == QKVLayout.BS3HD and self.max_seqlen_q != self.max_seqlen_kv:
             pytest.skip("BS3HD layout requires max_seqlen_q and max_seqlen_kv to be equal.")
 
-        if not is_fused_attn_kernel_available(
-                self.dtype, self.dtype, self.qkv_layout, self.attn_bias_type, self.attn_mask_type,
-                self.dropout_prob, self.num_heads_q, self.num_heads_kv, self.max_seqlen_q,
-                self.max_seqlen_kv, self.head_dim):
+        self.backend = FusedAttnHelper(
+            self.dtype, self.dtype, self.qkv_layout.value, self.attn_bias_type.value,
+            self.attn_mask_type.value, self.dropout_prob, self.num_heads_q, self.num_heads_kv,
+            self.max_seqlen_q, self.max_seqlen_kv, self.head_dim).get_fused_attn_backend()
+        if self.backend == NVTE_Fused_Attn_Backend.NVTE_No_Backend:
             pytest.skip("Unsupported inputs combination or device compute capability.")
 
+        if self.bias_shape != BiasShape.BIAS_1HSS:
+            if self.attn_bias_type != AttnBiasType.POST_SCALE_BIAS:
+                pytest.skip("B1SS, BHSS and 11SS bias shapes require POST_SCALE_BIAS.")
+            elif self.attn_mask_type not in [AttnMaskType.NO_MASK, AttnMaskType.CAUSAL_MASK]:
+                pytest.skip("B1SS, BHSS and 11SS bias shapes are only supported for "
+                            "AttnMaskType.NO_MASK and AttnMaskType.CAUSAL_MASK.")
+            elif self.backend != NVTE_Fused_Attn_Backend.NVTE_F16_arbitrary_seqlen:
+                pytest.skip("B1SS, BHSS and 11SS bias shapes are only supported for "
+                            "the F16_arbitrary_seqlen backend.")
+
```

Comment:
The logic here should be integrated into [nvte_get_fused_attn_backend](https://github.com/NVIDIA/TransformerEngine/blob/main/transformer_engine/common/include/transformer_engine/fused_attn.h#L146) and return no_backend for the unsupported combinations.

Reply:
I tentatively agree with this. If we intend to support this arbitrary mask workaround as a proper feature, we should have an `AttnBiasLayout` as part of the backend check.

@cyanguwa perhaps we can revisit this once I clean up the CI failures?